tensorboard/webapp/widgets/histogram/BUILD

```diff
@@ -25,6 +25,7 @@ tf_ng_module(
     deps = [
         ":formatter",
         ":types",
+        "//tensorboard/webapp:tb_polymer_interop_types",
```

Comment:
Seems unused?

Reply:
Yeah, looks like I missed this, good catch.